packages/terra-grid/docs/README.md

```diff
@@ -17,11 +17,11 @@ This component requires the following peer dependencies be installed in your app
 | react | ^16.8.5 |
 | react-dom | ^16.8.5 |
 
-
```

Comment:
This file can be removed since it was added to https://github.com/cerner/terra-core/pull/2849/files#diff-3016df0383159457864895364e929c7d

Reply:
done here https://github.com/cerner/terra-core/pull/2849/commits/1b00fdde104dc8806eaed133e4ad015b061ddb69.